src/index.html

```diff
@@ -7,8 +7,88 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link
+    rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+        class="logo_image"
+        src="./images/logo.png"
+        alt="moyo logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+
+          <li class="nav_item">
+            <a 
+             href="#" 
+             class="nav_link is-active">
```

Comment:
если у тега только два атрибута, то можно оставлять их на одной строке
```suggestion
            <a href="#" class="nav_link is-active">
```